mwrpy/level1/write_lev1_nc.py

```diff
@@ -402,7 +403,7 @@ def _add_blb(brt: RpgBin, blb: RpgBin, hkd: RpgBin, params: dict) -> None:
     )
 
     for time_ind, time_blb in enumerate(blb.data["time"]):
-        # WHY IS THIS HERE?
+        # TBD: timestamp in .BLB files changed with newer software version
```

Comment:
Software version is not (yet) included in the binary files, so it might be difficult to write it in a config file or calibration API. I'll work on a site independent solution.